core/modules/views/views.install

```diff
@@ -149,6 +149,33 @@ function views_update_1004() {
   }
 }
 
+/**
+ * Set all grid style Views to use the deprecrated table if not set already.
+ */
+function views_update_1005() {
+  $active_storage = config_get_config_storage('active');
+  $names = $active_storage->listAll();
+
+  foreach ($names as $name) {
+    if (substr($name, 0, 11) != 'views.view.') {
```

Comment:
How about using something like:

```php
foreach (config_get_names_with_prefix('views.view.') as $config_name) {
  ... do  ...things
}
```

Reply:
ah yes, that's a better function. Thanks @klonos.